Source/Krypton Components/Krypton.Toolkit/Designers/Designers/KryptonCommandLinkButtonDesigner.cs

```diff
@@ -0,0 +1,47 @@
+﻿#region BSD License
+/*
+ *
+ *  New BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/LICENSE)
```

Comment:
As these had already been developed in the Ext kit, then the start date year should be that.

Reply:
Dates are now correct.